e2e/tests/attendance-tests.spec.ts

```diff
@@ -1,360 +1,278 @@
 import { expect, test } from "#e2e/fixtures.ts";
 
-test.describe("Attendance Module", () => {
-  const FIXED_DATE = "2025-01-23";
+test.only("Record attendance for one activity", async ({ page }) => {
+  await page.getByRole("navigation").getByText("Attendance").click();
+  await page.getByRole("button", { name: "Record" }).click();
 
-  test.beforeEach(async ({ page }) => {
-    await page.clock.install({ time: FIXED_DATE });
-  });
+  const dateField = page.getByLabel("Date");
+  await expect(dateField).toHaveValue("1/23/2025");
 
-  test("Record attendance for one activity", async ({ page }) => {
-    await page.getByRole("navigation").getByText("Attendance").click();
-    await page.getByRole("button", { name: "Record" }).click();
+  await dateField.fill("12/15/2024");
 
-    /*
-      Verify the date field displays the current date
-    */
-    const fixedDate = new Date(FIXED_DATE);
-    const formattedDateRegex = new RegExp(
-      `${fixedDate.getMonth() + 1}/${fixedDate.getDate()}/${fixedDate.getFullYear()}|` +
-        `${(fixedDate.getMonth() + 1).toString().padStart(2, "0")}/${fixedDate.getDate().toString().padStart(2, "0")}/${fixedDate.getFullYear()}`,
-    );
-    const dateField = page.getByLabel("Date");
-    await expect(dateField).toHaveValue(formattedDateRegex);
```

Comment:
We set the date to a fixed constant in the test setup so we know it and can just assert it directly:
```typescript
wait expect(dateField).toHaveValue("1/23/2025");
```
It’s totally okay to hard-code the values for now.